frontend/src/modules/Administration/components/MaintenanceViewer.js

```diff
@@ -0,0 +1,438 @@
+import {
+  Box,
+  Button,
+  Card,
+  CardHeader,
+  CircularProgress,
+  Dialog,
+  Divider,
+  Grid,
+  IconButton,
+  MenuItem,
+  TextField,
+  Typography
+} from '@mui/material';
+import React, { useCallback, useEffect, useState } from 'react';
+import { Article, CancelRounded, SystemUpdate } from '@mui/icons-material';
+import { LoadingButton } from '@mui/lab';
+import { Label } from '../../../design';
+import {
+  getMaintenanceStatus,
+  stopMaintenanceWindow,
+  startMaintenanceWindow
+} from '../../../services/graphql/MaintenanceWindow';
```

Comment:
In the frontend we agreed on using global imports for common resources and relative imports inside each module or component. 

Anything outside of the `modules` directory is considered common. Common submodules such as `services` (refers to services in `frontend/src/services`, `utils`, `design`... can be imported directly:


```
import { Label } from 'design';
import { useClient } from 'services';
import { SET_ERROR, useDispatch } from 'globalErrors';
```

In another comment I also talked about the layout, which is related to the imports.For the maintenance services we would import them from the module as relative imports (not sure about the amount of `../`)
```
import {
  getMaintenanceStatus,
  stopMaintenanceWindow,
  startMaintenanceWindow
} from '../../services';

```

Reply:
Addressed this in the next update! Thanks for the insight into the structure

Reply:
There are still some imports that need to be fixed (e.g. globalErrors, design) Basically, anything inside the top-level directories except for modules